pkg/appmanager/as3Manager_test.go

```diff
@@ -103,7 +103,7 @@ var _ = Describe("As3Manager Tests", func() {
 			defer server.Close()
 			// Use Client & URL from our local test server
 			api := As3RestClient{server.Client(), server.URL}
-			_, status := api.restCallToBigIP(method, route, template)
+			_, status := api.restCallToBigIP(method, route, template, true)
```

Comment:
```suggestion
			_, status := api.restCallToBigIP(method, route, template, false)
```
Let's keep this to false as it is the ideal state.